src/child/actions/window.js

```diff
@@ -61,6 +61,14 @@ export function resizeToCompact() {
     return dispatch => {
         dispatch(resizing());
         const compactWindowDimensions = configService.getCompactWindowDimensions();
+
+        fin.desktop.Window.getCurrent().updateOptions({
```

Comment:
These options will still be set if the user goes from compact view to expanded view, preventing them from changing the window size in expanded view. The user should be still be able to resize and maximise in expanded view.